docs/upgrade-guide.md

```diff
@@ -1,5 +1,11 @@
 # Upgrade Guide
 
+## Upgrading to v3.2
+
+- The latest mapbox-gl release requires stylesheet to be included at all times. See [Get Started](/docs/get-started/get-started.md) for information about styling.
```

Comment:
Would it be possible to code-wise inject a minimal JSX `<style>` sheet to avoid this requirement? 

It would be great to avoid this need, especially for code pen usage and small demos.